### ISSUE TITLE
Cannot Archive App with Extension that uses AFNetworking Xcode 7 Beta 3

### ISSUE THREAD

**Issue by TravisCrist** [CocoaPods/CocoaPods#3819]
Hello, 

Env: Xcode 7 Beta 3 - Target iOS 9, 0.38.beta.2

In [issue 3811](https://github.com/CocoaPods/CocoaPods/issues/3811) I reported an issue about not being able to execute a post install script for AFNetworking GCC_PREPROCESSOR_DEFINITIONS. I was able to resolve that issue as seen in the closed issue. 

Now when I go to archive I am back to a similar error as seen in [case 3738](https://github.com/CocoaPods/CocoaPods/issues/3738). An idea was that the problem was related to a space so I made a sample app to try out that theory. Even without a space the issue exists. To provide the best sample of my app the sample app includes 2 targets today extensions one with a space and one without a space. 

Its available for download from: [Test.zip](https://drive.google.com/open?id=0B-X6pYk0cNryNG11cVA4UnR5Zzg)

The errors now seen are, for TodayNoSpaces: 
```
PhaseScriptExecution Copy\ Pods\ Resources /Users/dragon/Library/Developer/Xcode/DerivedData/TEST-bwsvqtumqfncrqfgbnklqsfdycrx/Build/Intermediates/ArchiveIntermediates/TEST/IntermediateBuildFilesPath/TEST.build/Release-iphoneos/TodayNoSpaces.build/Script-AA39B286C578218F71AE1B63.sh
    cd /Volumes/Storage/Dropbox/Torrenting/AutoDL/TEST
    /bin/sh -c /Users/dragon/Library/Developer/Xcode/DerivedData/TEST-bwsvqtumqfncrqfgbnklqsfdycrx/Build/Intermediates/ArchiveIntermediates/TEST/IntermediateBuildFilesPath/TEST.build/Release-iphoneos/TodayNoSpaces.build/Script-AA39B286C578218F71AE1B63.sh

building file list ... done

sent 29 bytes  received 20 bytes  98.00 bytes/sec
total size is 0  speedup is 0.00
building file list ... done
rsync: push_dir#3 "/Users/dragon/Library/Developer/Xcode/DerivedData/TEST-bwsvqtumqfncrqfgbnklqsfdycrx/Build/Intermediates/ArchiveIntermediates/TEST/InstallationBuildProductsLocation" failed: No such file or directory (2)
rsync error: errors selecting input/output files, dirs (code 3) at /SourceCache/rsync/rsync-45/rsync/main.c(580) [receiver=2.6.9]
rsync: connection unexpectedly closed (8 bytes received so far) [sender]
rsync error: error in rsync protocol data stream (code 12) at /SourceCache/rsync/rsync-45/rsync/io.c(453) [sender=2.6.9]
Command /bin/sh failed with exit code 12
```

For Today Spaces: 
```
PhaseScriptExecution Copy\ Pods\ Resources /Users/dragon/Library/Developer/Xcode/DerivedData/TEST-bwsvqtumqfncrqfgbnklqsfdycrx/Build/Intermediates/ArchiveIntermediates/TEST/IntermediateBuildFilesPath/TEST.build/Release-iphoneos/Today\ Space.build/Script-47F98FE175ABA738D4204C19.sh
    cd /Volumes/Storage/Dropbox/Torrenting/AutoDL/TEST
    /bin/sh -c \"/Users/dragon/Library/Developer/Xcode/DerivedData/TEST-bwsvqtumqfncrqfgbnklqsfdycrx/Build/Intermediates/ArchiveIntermediates/TEST/IntermediateBuildFilesPath/TEST.build/Release-iphoneos/Today\ Space.build/Script-47F98FE175ABA738D4204C19.sh\"

building file list ... done

sent 29 bytes  received 20 bytes  98.00 bytes/sec
total size is 0  speedup is 0.00
building file list ... done
rsync: push_dir#3 "/Users/dragon/Library/Developer/Xcode/DerivedData/TEST-bwsvqtumqfncrqfgbnklqsfdycrx/Build/Intermediates/ArchiveIntermediates/TEST/InstallationBuildProductsLocation" failed: No such file or directory (2)
rsync error: errors selecting input/output files, dirs (code 3) at /SourceCache/rsync/rsync-45/rsync/main.c(580) [receiver=2.6.9]
rsync: connection unexpectedly closed (8 bytes received so far) [sender]
rsync error: error in rsync protocol data stream (code 12) at /SourceCache/rsync/rsync-45/rsync/io.c(453) [sender=2.6.9]
Command /bin/sh failed with exit code 12
```

Thanks for taking a look.

**Comment by neonichu** [comment]
Thanks for the example project, it appears that this is still an issue with 0.38.

**Comment by segiddins** [comment]
I believe there's on open PR that fixes this?

**Comment by neonichu** [comment]
@segiddins you're right, this is fixed by #3820 — I somehow was under the impression that PR was included in the 0.38 release, sorry.
